include/xgboost/c_api.h

```diff
@@ -348,30 +357,58 @@ XGB_DLL int XGDMatrixGetUIntInfo(const DMatrixHandle handle,
  * \brief get number of rows.
  * \param handle the handle to the DMatrix
  * \param out The address to hold number of rows.
+ * \param signers list of usernames of signing clients
+ * \param signature user_signature
+ * \param sig_len signature length
```

Comment:
Here and below for changed functions: I think the `\param` should be `sig_lengths` instead of `sig_len`